### PR TITLE
Shadow: Define "in the same shadow-including document"

### PR DESCRIPTION
**Pull request by hayatoito**
The definition is intended to be used in https://github.com/w3c/pointerlock/pull/4.

### CODE REVIEW

**Comment by hayatoito** [comment]
It looks bikeshed updated a lot of places in dom.html. Is it okay?

File: dom.bs [hunk 1]
```diff
@@ -1541,6 +1541,10 @@ itself.</p>
 <p>An <a for=/>element</a> is <dfn export>in a shadow-including document</dfn> if its
 <a>shadow-including root</a> is a <a>document</a>.
 
+<p>An object <var>A</var> and an object <var>B</var> are <dfn export>in the same shadow-including
+document</dfn> if both <var>A</var> and <var>B</var> are <a>in a shadow-including document</a> and
+<var>A</var> and <var>B</var> share the same <a>shadow-including root</a>.
```
Comment (annevk) [inline dom.bs]: Apparently "in a shadow-including document" is only for elements. Is that wrong or should this also be for elements only? Maybe "node" for both makes more sense?

File: dom.html [hunk 2]
```diff
@@ -6269,11 +6272,6 @@ <h3 class="no-num no-ref heading settled" id="index-defined-here"><span class="c
   <h3 class="no-num no-ref heading settled" id="index-defined-elsewhere"><span class="content">Terms defined by reference</span><a class="self-link" href="#index-defined-elsewhere"></a></h3>
   <ul class="index">
    <li>
-    <a data-link-type="biblio">[ECMASCRIPT]</a> defines the following terms:
```
Comment (domenic) [inline dom.html]: @tabatkins any idea why this xref disappeared?

File: dom.bs (hunk 1 above)
Comment (domenic) [inline dom.bs]: +1 node for both

**Comment by domenic** [comment]
> It looks bikeshed updated a lot of places in dom.html. Is it okay?

It looks to me like it's mostly updates to the syntax highlighting, but I commented on one troublesome reference removal. But it doesn't seem worth preventing merge. As long as you used the latest Bikeshed (either via the HTTP API or by pulling the latest from master and running `bikeshed update`) then these changes should not be a problem.

File: dom.html (hunk 2 above)
Comment (tabatkins) [inline dom.html]: Yeah, it's a bug on my side. Got it reported yesterday, will be fixing it shortly.

**Comment by hayatoito** [comment]
Thank you for the review.

It looks that the relevant discussion is on going here: https://github.com/whatwg/dom/issues/238.
I would like to wait for the conclusion there, then update this PR accordingly.

**Comment by hayatoito** [comment]
Let me close this PR since it is no longer valid, as per https://github.com/whatwg/dom/issues/238.

**Pull request closed without merging**